src/xdist/plugin.py

```diff
@@ -261,6 +266,7 @@ def worker_id(request):
     """Return the id of the current worker ('gw0', 'gw1', etc) or 'master'
     if running on the master node.
     """
+    # TODO: remove "master", ideally for a None
```

Comment:
You mean returning `None` instead of `"master"` or `"controller"`?

Reply:
Yes correct if xdist is not active, there is no wirker/controller split